package.json

```diff
@@ -47,31 +48,32 @@
     "typescript": "~3.9.7"
   },
   "peerDependencies": {
-    "@aws-cdk/core": "1.143.0"
+    "@aws-cdk/core": "1.163.1"
   },
   "dependencies": {
-    "@aws-cdk/aws-apigatewayv2": "1.143.0",
-    "@aws-cdk/aws-apigatewayv2-authorizers": "1.143.0",
-    "@aws-cdk/aws-apigatewayv2-integrations": "1.143.0",
-    "@aws-cdk/aws-certificatemanager": "1.143.0",
-    "@aws-cdk/aws-cloudfront": "1.143.0",
-    "@aws-cdk/aws-cloudwatch": "1.143.0",
-    "@aws-cdk/aws-cloudwatch-actions": "1.143.0",
-    "@aws-cdk/aws-dynamodb": "1.143.0",
-    "@aws-cdk/aws-ec2": "1.143.0",
-    "@aws-cdk/aws-ecr": "1.143.0",
-    "@aws-cdk/aws-ecr-assets": "1.143.0",
-    "@aws-cdk/aws-lambda": "1.143.0",
-    "@aws-cdk/aws-lambda-event-sources": "1.143.0",
-    "@aws-cdk/aws-lambda-nodejs": "1.143.0",
-    "@aws-cdk/aws-route53": "1.143.0",
-    "@aws-cdk/aws-route53-targets": "1.143.0",
-    "@aws-cdk/aws-s3": "1.143.0",
-    "@aws-cdk/aws-s3-deployment": "1.143.0",
-    "@aws-cdk/aws-sns": "1.143.0",
-    "@aws-cdk/aws-sns-subscriptions": "1.143.0",
-    "@aws-cdk/aws-sqs": "1.143.0",
-    "@aws-cdk/core": "1.143.0",
+    "@aws-cdk/aws-apigatewayv2": "1.163.1",
+    "@aws-cdk/aws-apigatewayv2-authorizers": "1.163.1",
+    "@aws-cdk/aws-apigatewayv2-integrations": "1.163.1",
+    "@aws-cdk/aws-certificatemanager": "1.163.1",
+    "@aws-cdk/aws-cloudfront": "1.163.1",
+    "@aws-cdk/aws-cloudwatch": "1.163.1",
+    "@aws-cdk/aws-cloudwatch-actions": "1.163.1",
+    "@aws-cdk/aws-cloudfront-origins": "1.163.1",
```

Comment:
So this is the new dependency?

Reply:
Oh this is a new one to support the change over to the new class - it shouldn't technically be in this PR as its not used yet...

Reply:
Leaving this for now as its not much harm being there.